packages/static-cs-lite/lib/util/imageFrame/convert/color/convertRGBColorByPixel.js

```diff
@@ -9,11 +8,7 @@ const assertArrayDivisibility = require("../../../assertArrayDivisibility");
  * @returns {void}
  */
 function converter(imageFrame, rgbaBuffer) {
-  if (!assertArrayDivisibility(imageFrame, 3, ["decodeRGB: rgbBuffer must not be undefined", "decodeRGB: rgbBuffer length must be divisble by 3"])) {
```

Comment:
Array divisibility isn't actually a good test, since odd length images will always have even length representations, and there are other reasons why they sometimes don't get the expected length.